docs/framework.md

```diff
@@ -1151,130 +1121,32 @@ Information on the eligibility criteria for participants in the pre-qualificatio
 
 ```eval_rst
 .. jsoninclude-flat:: examples/full.json
-  :jsonpointer: /releases/7/preQualification
+  :jsonpointer: /releases/7/tender
   :include_only: eligibilityCriteria
   :ignore_path: /releases/7/
 ```
 
 </div>
 
-#### II.1.6. RFQ documents 
+#### II.1.6. Request for Qualification documents
 
-RFQ documents
+Request for Qualification (RFQ):
 
 > The set of documents issued by the procuring authority that constitute the basis of the qualification and potentially the pre-selection of candidates (the short list). Qualified (or short-listed candidates) will then be invited to submit a proposal (or to enter into a new phase prior to bid submission, such as a dialogue phase or interactive phase). ([Source](https://ppp-certification.com/ppp-certification-guide/glossary))
 
-Links to these documents should be provided using [Document](reference/documents) objects in the [preQualification/documents](../reference/schema/#release-schema.json,,preQualification/documents) array. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.
-
-**Example**: See [section I.4](#i-4-project-economic-and-social-benefits) for JSON and flattened examples of the `documents` building block.
-
-### II.2. List of pre-qualified suppliers 
+In a single-stage procedure, the RFQ documents are embedded in the Request for Proposal documents.
```

Comment:
What is the source of this statement? I wasn't aware that an RFQ could be part of an RFP; typically these are separate stages/processes.

Reply:
See the entry for Request For Proposal in https://ppp-certification.com/ppp-certification-guide/glossary

Reply:
Aha. NIGP describes RFP and RFQ as separate sub-processes: https://www.nigp.org/resource/global-best-practices/Public-Private%20Partnership%20P3%20Facilities%20and%20Infrastructure%20Best%20Practice.pdf?dl=true

What APMG above is saying is that qualification/selection criteria are included in the RFP documents. I think it's clearer to say that, than to say "RFQ documents are embedded in the RFP documents".

Reply:
Or, this statement from APMG for "One-stage tender" is clearer:

> A tender process in which the government issues its Request for Qualifications (RFQ) and Request for Proposal (RFP) together, and bidders submit their qualifications and bid as a single submission.

The original statement from AMPG for "Request for Proposal (RFP)" was:

> In one-stage open tender processes, the contents of a Request for Qualifications (RFQ) are embedded in the same set of documents, for example, the Request for Proposal (RFP) also includes the RFQ regulations.

This is somewhat different, because "documents embedded in documents" doesn't make sense, but "the contents of a Request for Qualifications (RFQ) are embedded", or "the Request for Proposal (RFP) also includes the RFQ regulations" does make sense.

Reply:
Good point. I updated the RFQ guidance in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/475291c062d8b7ed4c3179258825bd557fba155e and added guidance for the RFP, which was missing from the revised guidance, in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/7a95a3c70b5e7b40b5193b2e90cb962cd09842f6.